### PR TITLE
New blog and July 2020 activity report

### PR DESCRIPTION
**Pull request by cwross**
In an effort to take some of the administrative load away from @pulkomandy, I have made an attempt at writing up the activity report for July. Please do share feedback if any improvements or corrections are needed.

### CODE REVIEW

**Comment by pulkomandy** [comment]
Hi!

Line 27 (ARM bootloader): it got better, the bootloader is running again! Now they're working on loading the kernel and getting early debug output from there.

Line 56: the change for AVX-512 registers didn't work, I should revisit this. Maybe it's better to not mention it yet.

Line 97: I disagree with humdinger. My changes came first (and were just applying the last remaining suggestion from a ticket opened by Janus last year). I missed Mail and Emir fixed that.

Thanks for taking care of the report :)

**Comment by humdingerb** [comment]
> Line 97: I disagree with humdinger. My changes came first (and were just applying the last remaining suggestion from a ticket opened by Janus last year). I missed Mail and Emir fixed that.

I wasn't aware of that and stand corrected.

File: content/blog/dogcow/2020-08-01_july_2020_activity_report.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+Humdinger fixed bug #16321 in the ShowImage application. The timer now responds properly to user input.
+
+Andrew Lindesay added support for HaikuDepot Server repo meta-data to be artificially matched against multiple repos. Improvements were also made in HaikuDepot's logging mechanism to de-clutter the code and enable better systematic logging.
+
+pulkomandy fixed bug #8702, allowing the system to better distinguish video Ogg files from audio.
+
+## app_server
+
+Extra Bview drawing modes required for WebKit canvas support were implemented by Kacper Kasper.
```
Comment (KapiX) [inline content/blog/dogcow/2020-08-01_july_2020_activity_report.md:23]: KapiX (I don't really care all that much though)

File: content/blog/dogcow/2020-08-01_july_2020_activity_report.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+author = "dogcow"
+date = "2020-08-01 15:53:09-06:00"
+tags = []
++++
+
+Welcome to the July, 2020 Activity Report for the Haiku project!
+
+This report covers hrev54370 through hrev54484.
+
+## Applications
```
Comment (KapiX) [inline content/blog/dogcow/2020-08-01_july_2020_activity_report.md:13]: I think TextSearch also deserves a mention: https://git.haiku-os.org/haiku/commit/?id=2775737e9d73769f19c02de8dd713840d1edb9f2

File: content/blog/dogcow/2020-08-01_july_2020_activity_report.md (hunk 1 above)
Comment (cwross) [reply content/blog/dogcow/2020-08-01_july_2020_activity_report.md]: Fixed, thanks!

File: content/blog/dogcow/2020-08-01_july_2020_activity_report.md (hunk 2 above)
Comment (cwross) [reply content/blog/dogcow/2020-08-01_july_2020_activity_report.md:13]: Added!

**Comment by cwross** [comment]
Thank you all for your feedback. I believe I've resolved everyone's comments in the last commit. As a relative newcomer to the project, and as someone without C/C++/developer experience, making sense of some commit logs and figuring out what to filter out and how to write about certain items can be challenging, so I appreciate your patience and guidance!

**Comment by humdingerb** [comment]
> As a relative newcomer to the project, and as someone without C/C++/developer experience, making sense of some commit logs and figuring out what to filter out and how to write about certain items can be challenging...

You're doing very well! :)  And reviewing a PR should catch the things you've missed or misunderstood. Thanks!

Next time, you may want to "git commit --amend" your corrections so we don't have to squash several commits. (Though, since github offers to do that, it's no real issue anymore...)

**Pull request merged**